manifests/deployment.yaml.template

```diff
@@ -48,14 +48,14 @@ spec:
     port: 443
     targetPort: 8080
   selector:
-    name: casbin 
+    name: casbin
 ---
 apiVersion: admissionregistration.k8s.io/v1
 kind: ValidatingWebhookConfiguration
 metadata:
   name: casbin
 webhooks:
-  - name: casbin.org
+  - name: casin.casbin.org
```

Comment:
what's this?

Reply:
change `casin.casbin.org` to `webhook.casbin.org`